src/transformers/trainer.py

```diff
@@ -386,6 +395,37 @@ def setup_wandb(self):
                     self.model, log=os.getenv("WANDB_WATCH", "gradients"), log_freq=max(100, self.args.logging_steps)
                 )
 
+    def setup_comet(self):
+        """
+        Setup the optional Comet.ml integration.
+
+        Environment:
+            COMET_MODE:
+                (Optional): str - "OFFLINE", "ONLINE", or "DISABLED"
+            COMET_PROJECT_NAME:
+                (Optional): str - Comet.ml project name for experiments
+            COMET_OFFLINE_DIRECTORY:
+                (Optional): str - folder to use for saving offline experiments when `COMET_MODE` is "OFFLINE"
+
+        For a number of configurable items in the environment,
+        see https://www.comet.ml/docs/python-sdk/advanced/#comet-configuration-variables
```

Comment:
```suggestion
        see `here <https://www.comet.ml/docs/python-sdk/advanced/#comet-configuration-variables>`__
```
for a cleaner link in the rendered docs.